red_black_tree/rb_tree_tests.py

```diff
@@ -0,0 +1,2210 @@
+import unittest
+import random
+from datetime import datetime
+
+from rb_tree import RedBlackTree, Node, BLACK, RED, NIL
+NIL_LEAF = RedBlackTree.NIL_LEAF
+
+
+class RbTreeTests(unittest.TestCase):
+
+    def test_find_node(self):
+        """ Use the tree we get from the test_build function
+            and test the find function on each node"""
+        rb_tree = RedBlackTree()
+        rb_tree.add(2)
+        node_2 = rb_tree.root
+        rb_tree.add(1)
+        node_1 = rb_tree.root.left
+        rb_tree.add(4)
+        node_4 = rb_tree.root.right
+        rb_tree.add(5)
+        node_5 = node_4.right
+        rb_tree.add(9)
+        node_9 = node_5.right
+        rb_tree.add(3)
+        node_3 = node_4.left
+        rb_tree.add(6)
+        node_6 = node_9.left
+        rb_tree.add(7)
+        node_7 = node_5.right
+        rb_tree.add(15)
+        node_15 = node_9.right
+        """
+                            ___5B___
+                        __2R__      7R
+                      1B     4B    6B 9B
+                            3R         15R
+        """
```

Comment:
@Enether Weird. Maybe you should add a blank line between this block comment and the next single line comment.